addons/mail/static/src/composer/navigable_list.js

```diff
@@ -101,7 +89,7 @@ export class NavigableList extends Component {
     }
 
     navigate(direction) {
-        const activeOptionId = this.state.activeOption ? this.state.activeOption.id : -1;
+        const activeOptionId = this.state.activeIndex ? this.state.activeIndex : -1;
```

Comment:
Some missing occurrences of `state.options` not replaced by `props.options` below